account_statement_import_sheet_file/models/account_statement_import_sheet_parser.py

```diff
@@ -0,0 +1,501 @@
+            if not amount:
+                amount = abs(_decimal("amount_debit_column", values) or 0)
+            if not amount:
+                amount = -abs(_decimal("amount_credit_column", values) or 0)
+
+            balance = (
+                self._get_values_from_column(values, columns, "balance_column")
+                if columns["balance_column"]
+                else None
+            )
+            original_currency = (
+                self._get_values_from_column(
+                    values, columns, "original_currency_column"
+                )
+                if columns["original_currency_column"]
+                else None
+            )
+            original_amount = (
+                self._get_values_from_column(values, columns, "original_amount_column")
+                if columns["original_amount_column"]
+                else None
+            )
+            debit_credit = (
+                self._get_values_from_column(values, columns, "debit_credit_column")
+                if columns["debit_credit_column"]
+                else None
+            )
+            transaction_id = (
+                self._get_values_from_column(values, columns, "transaction_id_column")
+                if columns["transaction_id_column"]
+                else None
+            )
+            description = (
+                self._get_values_from_column(values, columns, "description_column")
+                if columns["description_column"]
+                else None
+            )
+            notes = (
+                self._get_values_from_column(values, columns, "notes_column")
+                if columns["notes_column"]
+                else None
+            )
+            reference = (
+                self._get_values_from_column(values, columns, "reference_column")
+                if columns["reference_column"]
+                else None
+            )
+            partner_name = (
+                self._get_values_from_column(values, columns, "partner_name_column")
+                if columns["partner_name_column"]
+                else None
+            )
+            bank_name = (
+                self._get_values_from_column(values, columns, "bank_name_column")
+                if columns["bank_name_column"]
+                else None
+            )
+            bank_account = (
+                self._get_values_from_column(values, columns, "bank_account_column")
+                if columns["bank_account_column"]
+                else None
+            )
+
+            debit_column = (
+                self._get_values_from_column(values, columns, "amount_debit_column")
+                if columns["amount_debit_column"]
+                else None
+            )
+
+            credit_column = (
+                self._get_values_from_column(values, columns, "amount_credit_column")
+                if columns["amount_credit_column"]
+                else None
+            )
+
+            if currency != currency_code:
```

Comment:
```suggestion
            if currency.lower() != currency_code.lower():
```
Avoid skipping lines just because 'eur' != 'EUR'
Ref.: https://github.com/OCA/bank-statement-import/pull/764